src/test/java/org/prebid/server/handler/openrtb2/AmpHandlerTest.java

```diff
@@ -394,30 +393,6 @@ public void shouldRespondWithCustomTargetingIncluded() {
                 + "\"hb_cache_id_bidder1\":\"value2\"}}"));
     }
 
-    @Test
```

Comment:
Pls keep this test, just change assertion.

Reply:
Are you sure? I have changed assertion and I thought that test made no sense to me. I might be wrong though.